README.md

```diff
@@ -1,21 +1,33 @@
 # llama-stack-operator
 This repo hosts a kubernetes operator that is responsible for creating and managing [llama-stack](https://github.com/meta-llama/llama-stack) server.
 
-### Table of contents
 
+## Features
+
+- Automated deployment of Llama Stack servers
+- Support for multiple [distributions](https://github.com/meta-llama/llama-stack?tab=readme-ov-file#distributions) (Ollama, vLLM)
```

Comment:
- Support for multiple [distributions](https://github.com/meta-llama/llama-stack?tab=readme-ov-file#distributions) (including Ollama, vLLM, and others)

Reply:
It is a nit, though.

Reply:
Updated

Reply:
Thank!